packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/ColumnActions.7.doc.mdx

```diff
@@ -0,0 +1,16 @@
+import ColumnActions from './ColumnActions?dev-site-example';
+
+# Column Actions
+
+### Description
+
+This example demonstrates a [Worklist Data Grid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) that supports Column Actions feature.
```

Comment:
can we reword this a little:

```suggestion
This example demonstrates a [Worklist Data Grid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) that supports Column Actions.
```

also not sure "Column Actions" needs to be capitalized.